lib/smart_proxy_pulp_plugin/pulp_node_plugin.rb

```diff
@@ -1,7 +1,17 @@
 module PulpMasterProxy
   class Plugin < ::Proxy::Plugin
     plugin "pulpnode", ::PulpProxy::VERSION
-    default_settings :pulp_url => 'https://localhost/pulp'
+    default_settings :pulp_url => 'https://localhost/pulp',
+                     :monitor_disks => {
+                       :pulp_dir => '/var/lib/pulp',
+                       :pulp_content_dir => '/var/lib/pulp/content',
+                       :mongodb_dir => '/var/lib/mongodb'
```

Comment:
Since there's a need to disambiguate dirs that are being monitored, presumably b/c these dirs are being handled differently, I’d argue that they can be left as separate properties. I don't think such a grouping is that useful, in fact, it can be problematic: the formatting has to be correct, it's not immediately clear which of the dirs inside the hash are required properties and which aren't. 

As a note, should you switch to flat properties, 'validate_readable' will work without any changes.